pat-static/testFiles

```diff
@@ -1 +0,0 @@
-../testFiles/
```

Comment:
Turns out PatternDataManager doesn't work either because of this... Though it's prolly better to find out how to set up junit to correctly set the root of "testFiles"

Reply:
Please see the other review below which comments on some code changes that will fix your bug issues.